### ISSUE TITLE
Keyboard focus is not visible on already selected/Focused controls in 'Comp terms' PDF.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#17]
Attach (recommended) or Link to PDF file here: https://mozilla.github.io/pdf.js/web/vie wer.html.

Configuration:
OS: Windows 11 Enterprise Insider Preview
Application version: Version 2312 (Build 17126.20032)

Steps to reproduce the problem:
1. Open URL:  https://mozilla.github.io/pdf.js/web/viewer.html
2. Navigate through the selected controls.
3. Observe whether Keyboard focus is visible on already selected/Focused controls in 'Comp terms' PDF or not.

What is the expected behavior? (add screenshot)
Focus should be clearly visible when element receives focus, even when it is already activated/Selected.

What went wrong? (add screenshot)
Keyboard focus is not visible on already selected/Focused controls (Toggle sidebar, Checkboxes under 'Find in document, Tools (Along with their list items) in 'Comp terms' PDF.

Users can't identify the focus on non-selected/focused controls.

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):
 https://mozilla.github.io/pdf.js/web/vie wer.html.

Attachments:

https://github.com/gdh1995/pdf.js/assets/93735775/40167b91-8079-4c25-ad52-321ca9aa586f

https://github.com/gdh1995/pdf.js/assets/93735775/dd26fc65-1255-49f6-b55e-3e83e6696c1b

**Comment by msftedad** [comment]
Hi Team,Any update on this issue?

**Comment by msftedad** [comment]
Hi @gdh1995, Any update on this issue? Please provide the ETA to fix this issue.
